torch/serialization.py

```diff
@@ -978,7 +1004,7 @@ def persistent_id(obj):
             # If storage is allocated, ensure that any other saved storages
             # pointing to the same data all have the same dtype. If storage is
             # not allocated, don't perform this check
-            if storage.data_ptr() != 0:
+            if str(storage.device) != "meta" and storage.data_ptr() != 0:
```

Comment:
Using `str(storage.device) != "meta"` to avoid the deprecation message mentioned above when accessing FakeTensor data_ptr

Reply:
xla IIRC

Reply:
ok, think this fix is good then to avoid accessing data_ptr specifically for FakeTensor storage